.env.example

```diff
@@ -4,6 +4,11 @@
 # However it should be easy to get going in development too. If you see an
 # uncommented option that means it's either mandatory to set or it's being
 # overwritten in development to make your life easier.
+#
+#
+# Set this up to handle Github App configuration 
+GITHUB_APP_ID=12345 
+GITHUB_CERT_PATH=/path/to/cert/pem
```

Comment:
Please change it to reflect the correct cert env variable.

If there is no default to set in the example then prepend the env vars with a hash.

I don't think there is any default we can provide for this.